dashboard/app/models/census/census_summary.rb

```diff
@@ -26,6 +26,72 @@ class Census::CensusSummary < ApplicationRecord
     HISTORICAL_YES: "HY",
     HISTORICAL_NO: "HN",
     HISTORICAL_MAYBE: "HM",
+    EXCLUDED: "E",
+    UNKNOWN: "U",
   }.freeze
   enum teaches_cs: TEACHES
+
+  # Use the zero byte as the quote character to allow importing double quotes
+  #   via http://stackoverflow.com/questions/8073920/importing-csv-quoting-error-is-driving-me-nuts
+  CSV_IMPORT_OPTIONS = {col_sep: ",", headers: true, encoding: 'bom|utf-8'}.freeze
+
+  # Seeds all the data from the S3 file.
+  # @param options [Hash] Optional map of options.
+  def self.seed_all(options = {})
+    options[:stub_school_data] = CDO.stub_school_data unless options.key?(:stub_school_data)
+
+    unless options[:stub_school_data]
```

Comment:
do you have a use in mind for this option? if not, I think it is ok to use `CDO.stub_school_data` directly here and get rid of the options

Reply:
Oh I was just trying to match `school.rb`. Sounds good!